ts/session/apis/open_group_api/opengroupV2/OpenGroupManagerV2.ts

```diff
@@ -163,25 +163,32 @@ export class OpenGroupManagerV2 {
     // here, the convo does not exist. Make sure the db is clean too
     await OpenGroupData.removeV2OpenGroupRoom(conversationId);
 
-    const room: OpenGroupV2Room = {
-      serverUrl,
-      roomId,
-      conversationId,
-      serverPublicKey,
-    };
-
     try {
+      const room: OpenGroupV2Room = {
+        serverUrl,
+        roomId,
+        conversationId,
+        serverPublicKey,
+      };
       // save the pubkey to the db right now, the request for room Info
       // will need it and access it from the db
       await OpenGroupData.saveV2OpenGroupRoom(room);
+
       const roomInfos = await openGroupV2GetRoomInfoViaOnionV4({
         serverPubkey: serverPublicKey,
         serverUrl,
         roomId,
       });
-      if (!roomInfos) {
+
+      if (!roomInfos || !roomInfos.id) {
         throw new Error('Invalid open group roomInfo result');
       }
+      await OpenGroupData.removeV2OpenGroupRoom(conversationId);
```

Comment:
So we make one attempt using the conversationId and if that doesn't work we fallback to the roomId?

Reply:
Resolved 👍